src/rhsm/config.py

```diff
@@ -105,7 +106,7 @@ def in_container():
     """
     # For development in containers we must be able to turn container detection
     # off
-    if os.environ.get("SMDEV_CONTAINER_OFF", False):
+    if bool(util.strtobool(os.getenv("SMDEV_CONTAINER_OFF", "False"))):
```

Comment:
I'm not against, but it has potential to break something. We will discuss this in the team.

Reply:
OK, I just simplified it without distutils.